doc/en/user/source/data/app-schema/joining.rst

```diff
@@ -83,3 +83,14 @@ Much like joining support, native encoding of nested filters is turned on by def
 Or, alternatively, by setting the value of the Java System Property "app-schema.encodeNestedFilters" to "false", for example ::
 
      java -DGEOSERVER_DATA_DIR=... -Dapp-schema.encodeNestedFilters=false Start
+
+UNION performance improvement for OR conditions
+-----------------------------------------------
+
+Is known that OR queries are difficult to optimize for postgresql and are usually slow.  App-Schema improve OR condition performance using UNION clauses instead OR for nested filter subqueries.

```

Comment:
I would reword this section and provide a concrete example, this what a an OR query looks like and this is what a UNION query looks like.